admin/osf_groups/serializers.py

```diff
@@ -0,0 +1,14 @@
+
+
+def serializer_group(osf_group):
+
+    return {
+        'named_id': osf_group._id,
+        'name': osf_group.name,
+        'created': osf_group.created,
+        'modified': osf_group.modified,
+        'creator': osf_group.creator,
+        'managers': list(osf_group.managers.all()),
+        'members': list(osf_group.members.all()),
```

Comment:
Also, `osf_group.members_only` return members who are not managers.  `members` returns all users in the group, members and managers.